### PR TITLE
Add Crossgen2 scenario to benchmarks

### PR DESCRIPTION
**Pull request by nattress**
Add the aggressively linked, IBC-tuned, version bubbled crossgen2
compilation scenario to the benchmark lab runs.

### CODE REVIEW

**Comment by nattress** [comment]
@rynowak PTAL

File: src/Benchmarks/link-a-thon.api-template.json [hunk 1]
```diff
@@ -74,5 +74,8 @@
     "Source": {
       "Project": "src/GrpcService/GrpcService.csproj"
     }
+  },
+  "LinkAThonTrimmedAndR2RSingleFileWithTrimListCrossgen2": {
```
Comment (rynowak) [inline src/Benchmarks/link-a-thon.api-template.json]: I'll update the zip and try this out before merging.

Comment (sebastienros) [reply src/Benchmarks/link-a-thon.api-template.json]: Try to create names that are more concise. Like `LinkAThonTrimmedR2RSingleCrossgen2`

But technically it doesn't matter so it's just a question of preference.

**Comment by sebastienros** [comment]
Should add the scenario in the script too

**Pull request merged**